packages/base-controller/src/BaseControllerV1.ts

```diff
@@ -30,6 +30,8 @@ export interface BaseState {
 }
 
 /**
+ * @deprecated in favor of controller-messenger pattern implemented in BaseControllerV2
```

Comment:
There are other patterns that BaseControllerV2 implements that improve upon BaseControllerV1, not just the messenger pattern. I'm also wondering if we want to make this message a bit more formal so people don't gloss over it:

``` suggestion
 * @deprecated This class has been renamed to BaseControllerV1 and is no longer recommended for use for controllers. Please use BaseController (formerly BaseControllerV2) instead.
```

Reply:
Love this. Applied!